images/modsecurity/cloudbuild.yaml

```diff
@@ -0,0 +1,24 @@
+timeout: 10800s
```

Comment:
Should we add license header?

Reply:
Cloud build files don't have license header (no idea why)

https://github.com/kubernetes/ingress-nginx/blob/main/cloudbuild.yaml